hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1048,6 +1050,8 @@ public void testSetSnapshotSummary() throws Exception {
   @Test
   public void testNotExposeTableProperties() {
     Configuration conf = new Configuration();
+    long maxHiveTablePropertySize =
+        conf.getLong(HIVE_TABLE_PROPERTY_MAX_SIZE, HIVE_TABLE_PROPERTY_MAX_SIZE_DEFAULT);
```

Comment:
We seem to set the "iceberg.hive.table-property-max-size" conf to null, but in this line we also populate a variable for this purpose. I don't think that we need to se the conf anymore, since this is an input param into setSnapshotStats() now.
Can't you just set long 'maxHiveTablePropertySize = <some value>'?

Reply:
Fixed.